Assets/AI/Actions/ChooseFreePosition.cs

```diff
@@ -0,0 +1,49 @@
+using UnityEngine;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using RAIN.Action;
+using RAIN.Core;
+using RAIN.Representation;
+using RAIN.Navigation;
+
+[RAINAction("Choose Free Position")]
+public class ChooseFreePosition : RAINAction
+{
+
+  public Expression MoveDistance = new Expression();
+
+  public Expression StayOnGraph = new Expression();
+
+  public Expression MoveTargetVariable = new Expression();
+
+  private float _defaultMoveDistance = 10f;
```

Comment:
[MUST]
Don't use underscore in `Member Block`
https://github.com/backstreetwx/Pac-Man/wiki/Coding-Standard#member-block